docs/release-notes/next-fortuna/PT-165593709_pinned_env_in_noise

```diff
@@ -0,0 +1,2 @@
+* Introduces a pinned environment in State Channels noise protocol. This is
```

Comment:
So there is no versioning in the actual channel protocol. Plus there is no versioning [in the doc](https://github.com/aeternity/protocol/pull/343) - apart from the tag name of course. May I suggest using these release notes for detailing what changed in the protocol?
```suggestion
* Introduces a pinned environment in State Channels noise protocol (off-chain), by introducing a `block_hash` field in some of the messages. This is
```